tests/intern-browserstack.ts

```diff
@@ -0,0 +1,14 @@
+export * from './intern';
```

Comment:
Ah, I wondered if that might be case. Unfortunate that it means you can't run the tests without grunt-dojo2, though.